appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/PayaraMicro.java

```diff
@@ -1368,13 +1427,19 @@ private void scanArgs(String[] args) {
                                 + "  --disablePhoneHome Disables sending of usage tracking information\n"
                                 + "  --version Displays the version information\n"
                                 + "  --logToFile <file-path> outputs all the Log entries to a user defined file\n"
+                                + "  --accessLog <directory-path> Sets user defined directory path for the access log\n"
+                                + "  --accessLogFormat Sets user defined a log format for access log\n"
```

Comment:
I think the article "a" is redundant. I would change it to: "Sets user defined log format ..."